code/controllers/subsystem/lobotomy_corp.dm

```diff
@@ -248,3 +273,6 @@ SUBSYSTEM_DEF(lobotomy_corp)
 	next_ordeal = null
 	RollOrdeal()
 	return TRUE // Very sloppy, but will do for now
+
+/datum/controller/subsystem/lobotomy_corp/proc/FailSuppression()
```

Comment:
Doesn't have to be a proc in LC subsystem. Move it elsewhere, since it's just a timed round-end.